Project.toml

```diff
@@ -0,0 +1,12 @@
+name = "NMF"
+uuid = "6ef6ca0d-6ad7-5ff6-b225-e928bfa0a386"
+license = "MIT"
+version = "0.4.1"
```

Comment:
Please remove the version. It just gets annoying to keep it in sync with the tags and it isn't actually necessary.